filebeat/module/kafka/log/ingest/pipeline.json

```diff
@@ -45,7 +45,7 @@
     {
       "rename": {
         "field": "@timestamp",
-        "target_field": "beat.read_time"
+        "target_field": "read_timestamp"
```

Comment:
@tsg This seems to be a breaking change? Also saw there is no CHANGELOG for this PR.